app/event/send-enrich-request-event.js

```diff
@@ -0,0 +1,17 @@
+const raiseEvent = require('./raise-event')
+const getCorrelationId = require('../payment-request/get-correlation-id')
+
+const sendEnrichRequestEvent = async (paymentRequest, user) => {
+  const { paymentRequestId } = paymentRequest
+  const { correlationId } = await getCorrelationId(paymentRequestId)
+  const event = {
+    id: correlationId,
+    name: 'payment-request-enrich-request-event',
```

Comment:
I think `-event` on the end is a bit redundant

Reply:
removed